src/components/implementation/no_interface/vkernel/vk_server.c

```diff
@@ -0,0 +1,47 @@
+
+#include "vk_types.h"
```

Comment:
Using <> puts the power in the build system, which is generally what you want.  I believe that `-I.` is in the include path so it should find local files.  What do you think?